tests/nextflow.config

```
@@ -6,7 +6,9 @@
 
 // TODO nf-core: Specify any additional parameters here
 // Or any resources requirements
-params.modules_testdata_base_path = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/'
-params.pipelines_testdata_base_path = 'https://raw.githubusercontent.com/nf-core/test-datasets/refs/heads/metaval'
+params {
+    modules_testdata_base_path = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/'
+    pipelines_testdata_base_path = 'https://raw.githubusercontent.com/genomic-medicine-sweden/test-datasets/metaval/l'
```

What is this /l at the end of the path?
```suggestion
    pipelines_testdata_base_path = 'https://raw.githubusercontent.com/genomic-medicine-sweden/test-datasets/metaval/l'
```

---

That's the typo. I cannot commit suggestions